docker/driver-loader-buster/Dockerfile

```diff
@@ -1,10 +1,15 @@
 FROM debian:buster
 
+ARG FALCO_COMMIT_SHA
+ARG FALCO_VERSION
```

Comment:
There is already a `FALCO_VERSION` below. I'd move that one here.

Reply:
Done.